src/main/java/mate/academy/springboot/datajpa/model/dto/request/ProductRequestDto.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.springboot.datajpa.model.dto.request;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Positive;
+import java.math.BigDecimal;
+import lombok.Data;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@Validated
+public class ProductRequestDto {
+    @Positive
+    private Long id;
```

Comment:
can a product **request** DTO have an id? what if the user will send us that id in the request body? We need separate request and response DTO's for the user to not be able to send us some fields, like `id`, so let's remove it, and just add another parameter to the update service methods